.github/workflows/kickstart-tests.yml

```diff
@@ -227,17 +217,30 @@ jobs:
             exit 1
           fi
 
+      - name: Create Permian settings file
+        working-directory: ./permian
+        run: |
+          cat <<EOF > settings.ini
+          [kickstart_test]
+          kstest_local_repo=${{ github.workspace }}/kickstart-tests
+          [library]
+          directPath=${{ github.workspace }}/kickstart-tests/testlib
+          [github]
+          pull-request=${{ needs.pr-info.outputs.pr_number }}
+          repository=${{ github.repository }}
+          token=${{ secrets.GITHUB_TOKEN }}
+          EOF
+
       - name: Run kickstart tests in container
         working-directory: ./permian
         run: |
           sudo --preserve-env=TEST_JOBS \
           PYTHONPATH=${PYTHONPATH:-}:${{ github.workspace }}/tplib \
           ./run_subset --debug-log permian.log \
-            -o library.directPath="${{ github.workspace }}/kickstart-tests/testlib" \
-            -o workflows.dry_run=False \
-            -o kickstart_test.kstest_local_repo="${{ github.workspace }}/kickstart-tests" \
+            --settings settings.ini \
+            --override workflows.dry_run=False \
             --testcase-query '${{ steps.generate_query.outputs.query }}' \
-            run_event '{"type":"github.pr","bootIso":{"x86_64":"file://${{ github.workspace }}/images/boot.iso"},"kstestParams":{"platform":"${{ needs.pr-info.outputs.platform }}"}}'
+            run_event '{"type":"everything","everything_testplan":{"configurations":[{"architecture":"x86_64"}],"point_person":"rvykydal@redhat.com","reporting":[{"type":"github-pr","data":{"pr-check-name":"${{ env.STATUS_NAME }} ${{ needs.pr-info.outputs.launch_args }}","pr-check-summary":"Kickstart tests run on a pull request comment"}}]},"bootIso":{"x86_64":"http://download.fedoraproject.org/pub/fedora/linux/development/rawhide/Server/x86_64/os/images/boot.iso"},"kstestParams":{"platform":"${{ needs.pr-info.outputs.platform }}"}}'
```

Comment:
Would it be possible to split this long line into several smaller ones? It is a little hard to read.

Reply:
Sure, good point, I failed in that some time ago for some reason, but now it went good, updated.